Makefile

```diff
@@ -597,23 +597,23 @@ service-binaries: cmd/future/minimatch/minimatch$(EXE_EXTENSION) cmd/future/back
 tools-binaries: tools/certgen/certgen$(EXE_EXTENSION)
 
 cmd/future/backend/backend$(EXE_EXTENSION): internal/future/pb/backend.pb.go internal/future/pb/backend.pb.gw.go api/backend.swagger.json
-	cd cmd/future/backend; $(GO_BUILD_COMMAND)
+	cd cmd/future/backend; $(GO_BUILD_COMMAND) -ldflags='-s -w' .
```

Comment:
This is not self documenting and involves copy-paste. I'd alias this.
`SLIM_GO_BUILD = -ldflags='-s -w'` or something